pkg/oc/clusterup/required_images.go

```diff
@@ -26,6 +26,9 @@ var OpenShiftImages = Images{
 	{Name: "etcd", PullSpec: "quay.io/coreos/etcd:v3.2.24"},
 }
 
+// forcePull specifies whether the images should always be pulled.
+var forcePull bool
```

Comment:
no globals

Reply:
i will change it after https://github.com/openshift/origin/pull/21146 merges